docs/cloud/cloud-tables/cloud-table-upload-data.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Upload data
+layout: default
+parent: Manage tables
+grand_parent: Cloud
+nav_order: 2
+---
+
+# How do I upload data to FeatureBase Cloud?
+{: .no_toc }
+
+
+## Before you begin
+
+{% include /cloud/cloud-before-begin.md %}
+* [Create a database](/docs/cloud/cloud-databases/cloud-db-create)
+
+## Naming standards
+
+{% include /concepts/standard-naming-obj.md%}
+{% include /cloud-table/cloud-standard-naming-table.md %}
+
+## Step 1: Navigate to Upload Data
+
+| Click **Databases** > database name > **Upload Data**. |
+
+## Step 2: Select a file to upload
+
+1. Click **Upload a file**.
+2. Select a file from the file browser.
+3. Click **Done**.
+
+{: .note }
+Currently, FeatureBase only supports .csv files. <br/>
+FeatureBase assumes that the first line in the uploaded file is the header row. <br/>
+
+## Step 3: Select column type for each column
+
+{% include /cloud-table/cloud-summary-table-pk.md %}
+
+1. Click on the kebab menu at the right side of the row of the targeted column.
```

Comment:
I've heard this called kebab, hamburger and possibly even three lines.

It's simpler to use the icon which is found in the `/help-on-help/style-guide/material-icons.md` file

So in this case you'd say something like:

```
1. Click <span class="material-icons md-18">more_vert</span> on the column.
```